meshmode/mesh/__init__.py

```diff
@@ -848,6 +857,16 @@ class Mesh(Record):
 
     .. attribute:: nelements
 
+    .. attribute:: element_nr_base
```

Comment:
```suggestion
    .. attribute:: base_element_nrs
```
? (to convey plural and avoid confusion with existing attribute)

Reply:
Renamed in latest version.